backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/CAdapterGenerator.kt

```diff
@@ -1012,9 +1050,9 @@ internal class CAdapterGenerator(
     private var functionIndex = 0
     fun nextFunctionIndex() = functionIndex++
 
-    internal val kGetTypeFuncType =
+    internal val kGetTypeFuncType get() =
```

Comment:
`lazy`?

Reply:
Somewhat conflicts the idea of having `codegen` for a limited amount of time only.